pkg/gatherers/clusterconfig/gather_node_logs.go

```diff
@@ -105,24 +118,44 @@ func nodeLogString(ctx context.Context, req *rest.Request) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	defer in.Close()
+	defer func() {
+		_ = in.Close()
```

Comment:
Should we log the error?

Reply:
Yeah, this is probably a good idea. Let me fix it.